Training25/Program.cs

```diff
@@ -3,12 +3,18 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T05 branch.
 // ------------------------------------------------------------------------------------------------
-namespace Training25 {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+using static System.Console;
+
+namespace Training25;
+internal class Program {
+   static void Main (string[] args) {
+      // Displays multiplication table from 1 to 10.
+      for (int tableNum = 1; tableNum < 11; tableNum++) {
+         for (int mulNum = 1; mulNum < 11; mulNum++) WriteLine ($"{tableNum} * {mulNum,2} = {tableNum * mulNum,-2}");

```

Comment:
-2 is not necessary here. 
```suggestion
         for (int mulNum = 1; mulNum < 11; mulNum++) WriteLine ($"{tableNum,2} * {mulNum,2} = {tableNum * mulNum}");
```